### ISSUE TITLE
Slider ignore decimal vaues

### ISSUE THREAD

**Issue by nrevutsky** [angular-slider/ngx-slider#137]
When stepArray is used, decimal point is ignored for value/highValue.
`<ng5-slider [(value)]="value" [(highValue)]="highValue" [options]="options"></ng5-slider>`

> export class AppComponent  {
>   value: number = 13.5;
>   highValue: number = 128.3;
>   options: Options = {
>     floor: 0.1,
>     ceil: 250.2,
>     stepsArray:[{value:0.1},
>     {value:13},{value:13.2},{value:13.5},{value:13.8},{value:14},
>     {value:128}, {value:255}],
>     step: 0.1
>   };
> }

value will be selected 14, even 13.5 is in list.

**Comment by nrevutsky** [comment]
Problem is from v1.2.0, everything was fine in 1.1.14

**Comment by nikolaof** [comment]
Do we have any news on that? I'm facing the same issue with values within stepsArray and many decimals like :
[
    { value: 0.04984748902869988 },
    { value: 0.10206717095165567 },
    { value: 0.16697085399552702 },
]

**Comment by piotrdz** [comment]
The issues mentioned by @nrevutsky and @nikolaof are fixed by recent changes to address #128 - it essentially the same thing as #128.

The issue mentioned by @p-programowanie is a different one and has to do with input normalisation, which doesn't work properly when using `stepsArray`. A workaround is to disable it through `enforceStep` and `enforceRange` options.

I fixed it now so that it is working as expected and added `enforceStepsArray` as another option that focuses on controlling only this behaviour.

Merged to master, pending next release.
